Libraries/Azure.psm1

```diff
@@ -824,11 +824,27 @@ Function Generate-AzureDeployJSONFile ($RGName, $ImageName, $osVHD, $RGXMLData,
     $jsonFile = $azuredeployJSONFilePath
 
     if ($ImageName -and !$osVHD) {
-        $imageInfo = $ImageName.Split(' ')
-        $publisher = $imageInfo[0]
-        $offer = $imageInfo[1]
-        $sku = $imageInfo[2]
-        $version = $imageInfo[3]
+        #Allow for heterogeneous images to be used
+        if( $ImageName -Match(";") )
```

Comment:
We're using comma as delimiter for separating parameter values in LISAv2. Please change this to comma.